bot/tests/base_test.py

```diff
@@ -60,7 +60,7 @@ def assert_response_contains(self, response: Message, expected_fragments: List[s
         error_message = msg.missing_fragment("{fragment}", "{response}")
```

Comment:
L49: po co `continue`? Po prostu: 

`if not message.out and message.id > sent_message_id`

Continue jest jeszcze paskudniejszy od `break` i nie ma co się w to bawić bez powodu